### PR TITLE
Enforce package:pedantic lints

### PR DESCRIPTION
**Pull request by konstantinullrich**

### CODE REVIEW

**Comment by konstantinullrich** [comment]
@googlebot I signed it!

File: lib/src/parameters.dart [hunk 1]
```diff
@@ -285,17 +276,30 @@ class Parameter extends Parameters {
...
           '${jsonEncode(string)}.$message');
     }
   }
 
+  /// Get a parameter named [named] that matches [test], or the value of calling
+  /// [orElse].
+  ///
+  /// [type] is used for the error message. It should begin with an indefinite
+  /// article.
+  dynamic _getTyped(String type, bool Function(dynamic) test) {
```
Comment (kevmoo) [inline lib/src/parameters.dart]: please don't move this function around – makes viewing the diff more difficult

File: analysis_options.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Defines a default set of lint rules enforced for
+# projects at Google. For details and rationale,
+# see https://github.com/dart-lang/pedantic#enabled-lints.
+include: package:pedantic/analysis_options.yaml
```
Comment (kevmoo) [inline analysis_options.yaml]: Delete everything out of this file except this line

**Comment by konstantinullrich** [comment]
Greetings from Germany

@kevmoo i implemented the changes you requested

**Pull request merged**